latest_cov_build.py

```diff
@@ -13,6 +13,16 @@
     from urllib import urlretrieve
 
 
+def get_url(repos_url, download_url, is_gecko):
+    latest_version = requests.get(repos_url)
+    data = latest_version.json()
+    tag_name = data['tag_name']
+    if is_gecko == 1:
```

Comment:
This is a bit brittle since the function is supposed to be generic. Let's return `download_url` and `tag_name` and make the caller generate the name.